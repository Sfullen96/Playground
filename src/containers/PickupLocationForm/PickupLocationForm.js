/*
* PickupLocationForm component
* Renders the form where a user can
* search for a pickup location.
* */
import React, {Component} from "react";

class PickupLocationForm extends Component {
    render() {
        return (
            <div>
                <h1>PickupLocationForm</h1>
            </div>
        );
    }
}

export default PickupLocationForm;