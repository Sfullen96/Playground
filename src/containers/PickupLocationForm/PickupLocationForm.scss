.pickup-location-form {
  position: absolute;
  left: 0;
  right: 0;
  margin: 0 auto;
  background-color: #F3CE56;
  width: 500px;
  padding: 25px;
  top: 50px;
  h1 {
    margin-top: 0;
    font-weight: bold;
    font-size: 32px;
  }
  form {
    label {
      font-size: 14px;
      color: #444;
    }
    input[type=text] {
      -webkit-border-radius: 0;
      -moz-border-radius: 0;
      border-radius: 0;
      margin-bottom: 10px;
    }
    button {
      width: 200px;
      height: 60px;
      background-color: #006e3e;
      color: #fff;
      font-size: 28px;
    }
  }
}

/* Custom, iPhone Retina */
@media only screen and (min-width : 320px) {

}

/* Extra Small Devices, Phones */
@media only screen and (min-width : 480px) {

}

/* Small Devices, Tablets */
@media only screen and (min-width : 768px) {

}

/* Medium Devices, Desktops */
@media only screen and (min-width : 992px) {

}

/* Large Devices, Wide Screens */
@media only screen and (min-width : 1200px) {

}

/*==========  Non-Mobile First Method  ==========*/

/* Large Devices, Wide Screens */
@media only screen and (max-width : 1200px) {

}

/* Medium Devices, Desktops */
@media only screen and (max-width : 992px) {

}

/* Small Devices, Tablets */
@media only screen and (max-width : 768px) {

}

/* Extra Small Devices, Phones */
@media only screen and (max-width : 480px) {

}

/* Custom, iPhone Retina */
@media only screen and (max-width : 320px) {

}