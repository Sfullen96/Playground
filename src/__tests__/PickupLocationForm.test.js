import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import PickupLocationForm from '../containers/PickupLocationForm/PickupLocationForm';

Enzyme.configure({ adapter: new Adapter() });

/*
* Test that the PickupLocationForm renders
* and matches it's latest snapshot
* */
it('should render PickupLocationForm component and it should match the snapshot', () => {
    const component = shallow( <PickupLocationForm /> );
    expect(component).toMatchSnapshot();
});