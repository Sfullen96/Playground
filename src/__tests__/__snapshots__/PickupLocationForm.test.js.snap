// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render PickupLocationForm component and it should match the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <PickupLocationForm />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <h1>
        PickupLocationForm
</h1>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "PickupLocationForm",
      },
      "ref": null,
      "rendered": "PickupLocationForm",
      "type": "h1",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <h1>
          PickupLocationForm
</h1>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "PickupLocationForm",
        },
        "ref": null,
        "rendered": "PickupLocationForm",
        "type": "h1",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getSnapshotBeforeUpdate": true,
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
